src/stores/useHolochainStore.js

```diff
@@ -135,60 +123,8 @@ const makeUseHolochainStore = ({ installed_app_id, app_ws_url, is_hpos_served, h
         resolve()
       })
     },
-    async hposHolochainCall({
-      path,
-      headers: userHeaders = {},
-      params,
-      method = 'post',
-    }) {
-      const axiosConfig = {
-        headers: {
-          'Content-Type': 'application/json',
-          'Access-Control-Allow-Origin': '*'
-        }
-      }
-
-      const HPOS_API_URL = `${window.location.protocol}//${window.location.host}`
-      const pathPrefix = '/holochain-api/v1/'
-      const fullUrl = `${HPOS_API_URL}${pathPrefix}${path}`
-
-      const authToken = localStorage.getItem('authToken')
-
-      const headers = {
-        'X-Hpos-Auth-Token': authToken,
-        ...axiosConfig.headers,
-        ...userHeaders
-      }
-
-      let response
-
-      switch (method) {
-        case 'get':
-          response = await axios.get(fullUrl, { params, headers })
-          return response.data
-
-        case 'post':
-          response = await axios.post(fullUrl, params, { headers })
-          return response.data
-
-        case 'put':
-          response = await axios.put(fullUrl, params, { headers })
-          return response.data
-
-        case 'delete':
-          response = await axios.delete(fullUrl, { params, headers })
-          return response.data
-
-        default:
-          throw new Error(`No case in hposCall for ${method} method`)
-        }
-    },    
     async loadAgentKycLevel(_, __) {
-      if( !is_hpos_served) {
-        return null // raw holochain doesn't have a mechanism for retrieving agent's kyc
-      }
-      
-      const kycLevel = await this.hposHolochainCall({path: 'kyc', headers: {}, params: {}, method: 'get'})
+      const kycLevel = await hposHolochainCall({path: 'kyc', headers: {}, params: {}, method: 'get'})
```

Comment:
I know the name of this function is not part of this PR, but by the same logic as above, this function would be called `fetchAgentKycLevel`

Reply:
renamed